pootle/apps/pootle_statistics/views.py

```diff
@@ -49,40 +73,26 @@ def get_context_data(self, *args, **kwargs):
         )
 
 
-class TPContributorsJSON(ContributorsJSONMixin, TPBrowseView):
-    pass
-
-
-class TPStoreContributorsJSON(ContributorsJSONMixin, TPBrowseStoreView):
-    pass
-
-
-class LanguageContributorsJSON(ContributorsJSONMixin, LanguageBrowseView):
-    pass
-
+def create_stats_view_class(mixin_class, base_class):
```

Comment:
I believe it's a bit surprising for the reader this `create_stats_view_class()` factory, and yeah it's another layer of indirection which adds up to the existing implicitness of CBV's, so I'd lean towards leaving it explicit.

(By the way, on the topic of CBVs and implicitness, there's [a very detailed article by Luke Plant](http://lukeplant.me.uk/blog/posts/djangos-cbvs-were-a-mistake/), one of its original authors. It's a very good read with many valid points being raised.)